.travis.yml

```diff
@@ -1,3 +1,4 @@
+dist: bionic
```

Comment:
Why do we need bionic?

Reply:
Because `graph-tool` is not available for xenial anymore. Ok.

Reply:
Also an update from time to time is not bad :-)

Reply:
Yep. But it requires us to update. Whereas the travis default should auto-update. But they are quite slow to do it...